airflow/utils/sqlalchemy.py

```diff
@@ -429,6 +430,20 @@ def with_row_locks(query, session: Session, **kwargs):
         return query
 
 
+@contextlib.contextmanager
+def lock_rows(query: Query, session: Session) -> Generator[list[Any], None, None]:
+    """Lock database rows during the context manager block.
+
+    This is a convenient method for ``with_row_locks`` when we don't need the
+    locked rows.
```

Comment:
Can we maybe change up this docstring a bit? Saying we don't "need" the locked rows then yielding them out doesn't seem consistent. (or the other way)

Reply:
Or I can remove the yielded value? It’s not used, I just felt more like why not

Reply:
Yeah, I'm good either way.